### PR TITLE
Update dependencies from predictor-extended

### PR DESCRIPTION
**Pull request by SferaDev**
### :pushpin: References

-   **Issue:** Closes #?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,100 +10,95 @@
...
         "@dhis2/d2-i18n-generate": "1.2.0",
-        "@dhis2/d2-ui-core": "7.1.3",
-        "@dhis2/d2-ui-forms": "7.1.3",
-        "@dhis2/ui-core": "4.21.1",
-        "@dhis2/ui-widgets": "2.1.0",
-        "@material-ui/core": "4.11.2",
+        "@dhis2/ui": "6.9.0",
+        "@eyeseetea/d2-api": "1.8.5",
+        "@eyeseetea/d2-ui-components": "2.6.6",
+        "@eyeseetea/xlsx-populate": "4.1.0",
```
Comment (tokland) [inline package.json:20]: Do we need xlsx-populate?

Comment (SferaDev) [reply package.json:20]: Not really

Comment (SferaDev) [reply package.json:20]: Feel free to clean them up, as much as you can 😄

**Review by tokland — APPROVED** [review]
All good to me, just added some minor commits. We'll start from here for the new CPR project.

**Pull request merged**